elements/tuto-reader/tutorials/alternate-firmware/docs/installing-alternate-firmware.md

```diff
@@ -1 +1,183 @@
-Test
+## Installing Alternate Firmware: Coreboot, WinZent, and U-Boot
+
+This tutorial provides instructions for installing alternate firmware 
+on your MinnowBoard MAX or Turbot: Coreboot, WinZent, and U-Boot. 
+
+### Coreboot
+Coreboot (formerly known as LinuxBIOS) is a free and open source extended firmware 
+platform that delivers a fast and secure boot experience for embedded systems as an 
+alternative option to using proprietary BIOS firmware. In contrast to a conventional 
+BIOS, Coreboot initializes only the minimum required hardware (such as RAM, PCI, 
+serial) and then executes additional payload boot logic for the rest. This two-stage 
+approach helps achieve minimal boot times and a small footprint.
+
+With the separation of hardware initialization and later boot logic, Coreboot can do 
+the following: 
+
+- Scale from specialized applications that run directly from firmware 
+- Run operating systems in flash 
+- Load custom boot-loaders 
+- Implement firmware standards such as PC BIOS services or UEFI 
+
+By permitting systems the ability to include only the features necessary to run the 
+target application, code and flash space required are reduced. For further information, 
+refer to the [Coreboot website]( http://www.coreboot.org).
+
+The status on Coreboot's [Supported Motherboards](http://www.coreboot.org/Supported_Motherboards) 
+page indicates an unknown status for MinnowBoard boards; however, mainline coreboot 
+support for MinnowBoard boards (by Sage Electronic Engineering) does exist, as shown 
+on [review.coreboot.org](http://review.coreboot.org/gitweb?p=coreboot.git). Coreboot 
+supports booting from Serial ATA and USB 2.0/3.0. Booting from SD storage isn't working, 
+and S3 suspend/resume support isn't working either. 
+
+Below is a step-by-step guide to building coreboot for the Minnowboard MAX and Turbot.
+
+#### Install the dependencies
+
+  ``` 
+  $ sudo apt-get install gcc git make ncurses-dev flex bison
+  ```
+
+#### Get Coreboot 
+For simplicity, put all downloads and items extracted into the same directory.
+
+  ```
+ 	$ git clone http://review.coreboot.org/p/coreboot
```

Comment:
Bash steps should not be prefixed with `$` as that prohibits the ability to copy/paste the instructions. The `$` prefix should only be used when examples are provided that include application output and not expected for the user to replicate.